hostinit/setup_test.go

```diff
@@ -63,7 +63,7 @@ func TestSetupReadyHosts(t *testing.T) {
 				mock.MockInstances[id] = instance
 			}
 			Convey("when running setup", func() {
-				So(hostInit.setupReadyHosts(ctx), ShouldNotBeNil)
+				So(hostInit.setupReadyHosts(ctx), ShouldBeNil)
```

Comment:
Why should this be nil?

Reply:
When this test was written, an error returned from IsHostReady would cause the setupReadyHosts function to return an error early. Now, it just logs the error and continues trying to setup other hosts which is correct